src/components/Model3DViewer/stories/full.md

```diff
@@ -79,7 +79,7 @@ function ExampleComponent(props) {
 | `assetId`                  | id of asset to highlight                                            | `number`                                                                            |         |
 | `boundingBox`              | bounding box object, that describes dimension of viewed asset nodes | `THREE.Box3`                                                                        |         |
 | `cache`                    | object for caching 3D viewers instances                             | `{ [name:string]: any }`                                                            |         |
-| `enableKeyboardNavigation` | enable keyboard navigation in viewer                                | `boolean`                                                                           |  false  |
+| `enableKeyboardNavigation` | enable keyboard navigation in viewer. Viewer must be in focus       | `boolean`                                                                           |  false  |
```

Comment:
Could you set `enableKeyboardNavigation={true}` by default? @anderscognite was asking about that and I guess it is safe to use keyboard navigation from now.

Reply:
Done